specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/examples/ConsumerInvitations_Get.json

```diff
@@ -0,0 +1,36 @@
+{
+  "parameters": {
+    "location": "East US 2",
+    "api-version": "2020-05-04-preview",
+    "invitationId": "dfbbc788-19eb-4607-a5a1-c74181bfff03"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 13 Sep 2017 18:04:32 GMT",
+        "x-ms-request-id": "d5496da4-9c52-402f-b067-83cc9ddea888",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-ratelimit-remaining-tenant-reads": "14999",
+        "x-ms-correlation-request-id": "25c78f97-0b0a-4fe9-ad39-883a482265cd"
+      },
+      "body": {
+        "properties": {
+          "location": "eastus2",
```

Comment:
>"location": "eastus2" [](start = 10, length = 21)

this should be at the top level if the invitation is regional

Reply:
@pilor - I'm not sure if it matters but ListInvitation routing will be "ProxyOnly, Tenant, Fanout" while consumerInviations and rejectInvitaions routing will be "ProxyOnly, Tenant, LocationBased".

Can we use location at top level for a location based proxy resource ?